tests/copilot_helper/fixtures/make_addons/expected/web/addons/my-aurora-db.yml

```diff
@@ -0,0 +1,265 @@
+# Generated by copilot-helper v0.1-TEST / 2023-08-22 16:00:00
+Parameters:
+  # Copilot required Parameters...
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+
+Resources:
+  myAuroraDbLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub "${App}-${Env}-myAuroraDb-aurora-create-user"
```

Comment:
Function names must be unique. What about a situation where there are multiple databases of the same type? Should be  namespaced with service name as well.